hassio/src/addon-view/config/hassio-addon-audio.ts

```diff
@@ -171,6 +177,10 @@ class HassioAddonAudio extends LitElement {
   }
 
   private async _saveSettings(ev: CustomEvent): Promise<void> {
+    if (this.disabled) {
+      return;
+    }
```

Comment:
This is redundant but can't hurt I guess

Reply:
This is because of `ha-progress-button` it is styled like disabled. but you can still click it :see_no_evil: